developers/index.md

```diff
@@ -5,38 +5,37 @@ title: Developer Guide
 
 # Overview
 
-A variety of different parts of openHAB can be extended, we go through all of them.
+A variety of different parts of openHAB can be extended. We will cover the most important ones here.
```

Comment:
```suggestion
A variety of different parts of openHAB can be extended.
We will cover the most important ones here.
```

Please use one line per sentence.
The website build will take car of the correct presentation.